meson_options.txt

```diff
@@ -40,7 +40,7 @@ option('enable-blas', type: 'boolean', value: true)
 option('enable-fp16', type: 'boolean', value: false)
 option('enable-cublas', type: 'boolean', value: false)
 option('enable-openmp', type: 'boolean', value: true)
-option('enable-neon', type: 'boolean', value: false)
+option('enable-neon', type: 'boolean', value: true)
```

Comment:
How about enabling the `enable-neon` option when necessary?
It doesn't seem to be mandatory, so it may not always need to be set as true. 👍

Reply:
Yes. You are right; it seems better to make the default Meson option more conservative. I will update it. Thank you.